react/features/video-layout/middleware.web.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 
 import VideoLayout from '../../../modules/UI/videolayout/VideoLayout.js';
-import { CONFERENCE_JOINED, CONFERENCE_WILL_LEAVE } from '../base/conference';
+import { CONFERENCE_WILL_LEAVE } from '../base/conference';
+import { MEDIA_TYPE } from '../base/media/index.js';
```

Comment:
Weird import?

Reply:
Yep! Sorry! My editor is adding auto imports from time to time and I have missed to fix this one!

Will fix it!